module/icp/algs/sha2/sha2.c

```diff
@@ -78,7 +78,7 @@ static uint8_t PADDING[128] = { 0x80, /* all zeros */ };
  * barriers to reduce stack frame size. This can reduce the SHA512Transform()
  * stack frame usage from 3k to <1k on ARM32, for example.
  */
-#if defined(_ILP32) || defined(__powerpc)	/* small stack */
+#if defined (APPLE) || defined(_ILP32) || defined(__powerpc)	/* small stack */
```

Comment:
```
#if (defined(__APPLE__) && !defined(__OPTIMIZE__)) || defined(_ILP32) || defined(__powerpc)    /* small stack */
```

since when we use any optimization level other than 0 we get better code that does not trigger the frame-larger-than warning, and since we use \_\_APPLE\_\_ rather than APPLE everywhere else.

Reply:
Yeah that is fine, will fix.  APPLE was clearly just dumb.